stable/restore/README.md

```diff
@@ -150,11 +150,14 @@ The following tables list the configurable parameters of the restore chart and t
 
 | Parameter | Description | Default |
 | ----- | ----------- | ------ |
-| `restore.type` | What type of restore to perform: [ "database" | "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
-| `restore.target` | Where to restore `TO` | `{{ .Values.database.name }}` |
-| `restore.source` | Where to restore `FROM` [ backupset | url | `:latest` ] | `:latest` |
+| `restore.type` | What type of restore to perform: [ "database" \| "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
+| `restore.target` | Where to restore `TO` | `"demo"` |
+| `restore.source` | Where to restore `FROM` [ _backupset_ \| _url_ \| `:latest` \| `:group-latest` ] | `:latest` |
 | `restore.credentials` | Credentials to use for a URL source (user:password) | `""` |
 | `restore.autoRestart` | Whether to automatically restart the database and trigger the restore (true/false). Only valid for a "database" restore | `true` |
+| `restore.labels` | Process labels used to filter the complete set of archiveIds that should be restored which then defines the new state of the database upon restore. The setting works with NuoDB 4.2+ | `backup cluster0` |
```

Comment:
We should say that this value is interpreted as `<key> <value> [<key> <value> ...]`. I'm not sure what `backup cluster0` means. I would expect labels to look like `type backup`. Also, perhaps we should say how multiple key-value pairs interpreted, i.e. _all_ labels have to match as opposed to _any_.

Reply:
The label syntax in Helm is a bit different. It expects a dictionary of labels, like:
```
restore:
  labels:
    label1: value1
    label2: value2
...
```
I will update the default value as it implies that `<key> <value> [<key> <value>]` is used. 
The default value of `{ "backup": "cluster0" }` will select the archive ids of all HC SMs in `cluster0` to be restored. The `backup` label is used historically to define the so-called "backup group" (a group of HC SMs that are backed-up together using `nuodocker backup database`).